### DISCUSSION TITLE
Reaction Carbuncle in SU2_NEMO

### DISCUSSION THREAD

**Discussion by PreparingForLaunch** [su2code/SU2#2004] [Q&A]
SU2 Experts,

I'm running NEMO with MPP and I'm getting a Carbuncle in electron density and EV temperature. Does anyone have any suggestion on how to resolve?

Thanks.

**Comment by WallyMaier** [comment]
Hi,

Could you provide more information?  What are the conditions you are running at? numerical schemes?  what does your mesh look like?

**Reply by PreparingForLaunch** [reply]
I'm trying to run the RAM-C II at a Mach Number of 23.9. I have tried AUSM, AUSM+UP, and AUSM+M with and without the slope limiter. The mesh is 3-D and fully structured with a row of cells directly centered on the stagnation line. I have also tried 7 and 11 species air using Mutation++. On the development branch, I've tried adding viscosity, but that has the side-effect of having free electron diffusion upstream of the shock. 

The solution either oscillates or has a carbuncle, the only thing I've managed to do to get rid of it is increasing the standoff distance with an applied magnetic field. However, I still need to get a good baseline case.

**Reply by WallyMaier** [reply]
Hi  @PreparingForLaunch,

I apologize for the very long delay.  Have you found a solution to this?
If not, could you post a few screenshots of your solution?

**Reply by PreparingForLaunch** [reply]
No problem, I have not found a great solution. I added a limiter, but that didn't seem to do anything. I forced ambipolar diffusion and added viscosity, which helped a little. The biggest "fix" was only running on one node/processer, which means the MPI might be causing the issues, but I tried on 6 different machines with different modules and got the same problems.
![MPI_Example1_Density_0](https://github.com/su2code/SU2/assets/129337235/1f13fdf8-563b-4d73-82c1-3abc0fde197b)
![MPI_Example2_Density_0](https://github.com/su2code/SU2/assets/129337235/2a15628f-0fbd-4947-9d68-f45c9267c566)
